README.md

```diff
@@ -22,6 +22,7 @@ Shelley | [Design](https://hydra.iohk.io/job/Cardano/cardano-ledger/delegationDe
 Allegra & Mary | [Multi-Currency](https://eprint.iacr.org/2020/895 "Multi-Currency Ledgers"), [UTXOma](https://iohk.io/en/research/library/papers/utxoma-utxo-with-multi-asset-support/ "UTXOma:UTXO with Multi-Asset Support") | [Spec](https://hydra.iohk.io/job/Cardano/cardano-ledger/specs.shelley-ma/latest/download-by-type/doc-pdf/shelley-ma "A Formal Specification of the Cardano Ledger with a Native Multi-Asset Implementation") | [CDDL](https://github.com/input-output-hk/cardano-ledger/tree/master/eras/shelley-ma/test-suite/cddl-files)
 Alonzo | [eUTXO](https://iohk.io/en/research/library/papers/the-extended-utxo-model/ "The Extended UTXO Model")| [Spec](https://hydra.iohk.io/job/Cardano/cardano-ledger/specs.alonzo-ledger/latest/download-by-type/doc-pdf/alonzo-changes "A Formal Specification of the Cardano Ledger integrating Plutus Core") | [CDDL](https://github.com/input-output-hk/cardano-ledger/tree/master/eras/alonzo/test-suite/cddl-files)
 Babbage | [batch-verification](https://iohk.io/en/research/library/papers/on-uc-secure-range-extension-and-batch-verification-for-ecvrf/ "On UC-Secure Range Extension and Batch Verification for ECVRF"), [CIP-31](https://github.com/cardano-foundation/CIPs/pull/159 "Reference inputs"), [CIP-32](https://github.com/cardano-foundation/CIPs/pull/160 "Inline datums"), [CIP-33](https://github.com/cardano-foundation/CIPs/pull/161 "Reference scripts") | [Spec](https://hydra.iohk.io/job/Cardano/cardano-ledger/specs.babbage-ledger/latest/download-by-type/doc-pdf/babbage-changes "Formal Specification of the Cardano Ledger for the Babbage era") | [CDDL](https://github.com/input-output-hk/cardano-ledger/tree/master/eras/babbage/test-suite/cddl-files)
+Conway | | | [CDDL](https://github.com/input-output-hk/cardano-ledger/tree/master/eras/conway/test-suite/cddl-files)
```

Comment:
you don't have to make this change, but here are my thought:

* CIP-1694 can be in the "design docs" column. but we might want to link to the section that mentions the first stage of the implementation plan, so that it is clear that conway is not all of CIP-1694.
* We could put some kind of WIP link to the formal ledger for conway formal spec.